netaddr.go

```diff
@@ -359,7 +359,7 @@ func parseIPv6(in string) (IP, error) {
 // the standard library's ParseIP: https://play.golang.org/p/qdjylUkKWxl.
 // To convert a standard library IP without the implicit unmapping, use
 // FromStdIPRaw.
-func FromStdIP(std net.IP) (ip IP, ok bool) {
```

Comment:
please restore the names here. The bool is mysterious otherwise, and the docs then refer to a mysterious "ok" from line 354.

Reply:
I'm happy to update the docs accordingly, but isn't the comma-ok idiom common enough in Go where the meaning can be reasonably inferred without the names? Same for below.

Reply:
Really I guess these statements would apply to all the changes I've made here.

Reply:
I think 1 or 2 lines of this PR were fine.